src/core/models/directanswer.js

```diff
@@ -36,6 +38,33 @@ export default class DirectAnswer {
         },
         verticalConfigId: directAnswer.verticalKey
       }
-    });
+    };
+
+    const directAnswerFieldApiName = directAnswerData.answer.fieldApiName;
+    const formatterExistsForDirectAnswer = formatters && directAnswerFieldApiName in formatters;
+
+    if (formatterExistsForDirectAnswer) {
+      directAnswerData.answer.value = this._getFormattedValue(directAnswerData, formatters[directAnswerFieldApiName]);
+    }
+
+    return new DirectAnswer(directAnswerData);
+  }
+
+  /**
+   * Applies a formatter to a direct answer value
+   *
+   * @param {Object} data directAnswerData
+   * @param {function} formatter a field formatter to apply to the answer value field
+   */
+  static _getFormattedValue (data, formatter) {
+    if (!data.answer || !data.relatedItem) {
+      return null;
```

Comment:
If we can't apply formatters to the value, should we return `data` here?

Reply:
This function is supposed to return a string (i'll update the jsdoc), and I thought it made more sense to return a string, or null if the formatter can't be applied. Returning a string or an object if the formatter can't be applied seems like a strange interface to me